modules/nw-route-specific-annotations.adoc

```diff
@@ -70,6 +70,21 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 Environment variables cannot be edited.
 ====
 
+.Router timeout variables
+
+TimeUnits are represented by a number followed by the unit: us *(microseconds), ms (milliseconds, default), s (seconds), m (minutes), h *(hours), d (days).
+
+The regular expression is: [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
+[cols="3*"]
+|===
+| ROUTER_THREADS | 4s | Specifies the number of threads for the haproxy router.
+| ROUTER_DEFAULT_TUNNEL_TIMEOUT | 1h | Length of time for TCP or WebSocket connections to remain open. If you have websockets/tcp connections (and any time HAProxy is reloaded), the old HAProxy processes will stay for that period.
+| ROUTER_DEFAULT_CLIENT_TIMEOUT | 30s | Length of time that a client has to acknowledge or send data.
+| ROUTER_DEFAULT_SERVER_TIMEOUT | 30s | Length of time that a server has to acknowledge or send data.
+| RELOAD_INTERVAL | 5s | The minimum frequency the router is allowed to reload to accept new changes.
```

Comment:
seems some timeout variables are missing, e.g.
```
ROUTER_BACKEND_CHECK_INTERVAL
ROUTER_CLIENT_FIN_TIMEOUT
ROUTER_DEFAULT_CONNECT_TIMEOUT
ROUTER_DEFAULT_SERVER_FIN_TIMEOUT
ROUTER_SLOWLORIS_HTTP_KEEPALIVE
ROUTER_SLOWLORIS_TIMEOUT
ROUTER_METRICS_HAPROXY_TIMEOUT
```
please refer to `Router timeout variables` section in https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#env-variables

Reply:
Acked. Added all the timeout variables from 3.11 docs.

Reply:
Great! They look good now. Thank you!